mqtt/rust-toolchain.toml

```diff
@@ -0,0 +1 @@
+../rust-toolchain.toml
```

Comment:
Are the symlinks needed? Does rustup not look for it above the workspace root?

Reply:
It seems like rustup does look for it in the workspace root. I will remove the symlinks.